TheMET/TheMET/Services/FeaturedArtService.swift

```diff
@@ -136,10 +138,20 @@ class FeaturedArtService {
                     self?.isFeaturedArtLoading = false
                     return
                 }
-                self?.storeFeaturedArtData(artId: randomId, date: Date.now)
-                self?.artFileManager?.write(art: object)
-                self?.featuredArt = .loaded(object)
-                self?.isFeaturedArtLoading = false
+                guard let imageURL = object.primaryImage else {
+                    self?.forceUpdateFeaturedArt()
+                    return
+                }
+                self?.imageLoader.loadImage(urlString: imageURL, completion: { image in
```

Comment:
Are you sure you really need to download an actual image? You can just check that the `object.primaryImage` is valid